### PR TITLE
fix:document to html encoder added

### PR DESCRIPTION
**Pull request by alihassan143**
#156

### CODE REVIEW

**Comment by alihassan143** [comment]
@LucasXu0  pr ready for review

File: test/plugins/html/html_document_test.dart [hunk 1]
```diff
@@ -336,3 +342,206 @@ const rawHTML = """<h1>AppFlowyEditor</h1>
...
+                  '            \' \n'
+                  '\'\n'
+                  '            \' \n'
+                  '\'\n'
+                  '            \' \n'
+                  '\'\n'
+                  '            \' \n'
+                  '\'\n'
+                  '            \' '
+            }
```
Comment (LucasXu0) [inline test/plugins/html/html_document_test.dart:541]: Why are there so many line breaks ('\n')?

File: test/plugins/html/html_document_test.dart [hunk 2]
```diff
@@ -336,3 +342,206 @@ const rawHTML = """<h1>AppFlowyEditor</h1>
...
+        'type': 'numbered_list',
+        'data': {
+          'delta': [
+            {
+              'insert': '\n'
+                  '\'\n'
+                  '            \'     Select text to trigger to the toolbar to format your notes.\n'
+                  '\'\n'
+                  '            \'   '
+            }
```
Comment (LucasXu0) [inline test/plugins/html/html_document_test.dart:496]: same as the previous comment. the line break

**Comment by LucasXu0** [comment]
Hey, @alihassan143. I added some comment but I didn't check the logic yet.

**Comment by alihassan143** [comment]
> Hey, @alihassan143. I added some comment but I didn't check the logic yet.

i did some changes kinndly check

**Comment by alihassan143** [comment]
@LucasXu0 will you check the changes

**Comment by alihassan143** [comment]
@LucasXu0  i know their are redundant commits but converter was missing some attributes i fix them now kindly review the pr

**Pull request closed without merging**